### PR TITLE
dbsta: add cell area to metrics to mirror report information and totals

### PR DESCRIPTION
**Pull request by gadfort**
Adds area to metrics recorded with `report_cell_usage` and the totals for the module.

### CODE REVIEW

**Comment by gadfort** [comment]
@maliberty could you take a look at this?

File: src/dbSta/src/dbSta.cc [hunk 1]
```diff
@@ -536,17 +536,25 @@ void dbSta::report_cell_usage(odb::dbModule* module, const bool verbose)
...
+        = toLowerCase(regex_replace(type_name, regexp, "_"));
+
+    logger_->metric("design__instance__count__class:" + type_class,
                     stats.count);
+    logger_->metric("design__instance__area__class:" + type_class,
+                    stats.area / area_to_microns);
   }
+  logger_->metric("design__instance__count__class:total", total_usage);
+  logger_->metric("design__instance__area__class:total",
+                  total_area / area_to_microns);
```
Comment (maliberty) [inline src/dbSta/src/dbSta.cc:557]: We already have design__instance__count, do we need design__instance__count__class:total?  Likewise design__instance__area and design__instance__area__class:total

Comment (gadfort) [reply src/dbSta/src/dbSta.cc:557]: This provides the number for the selected modinst (design__instance__area is for the entire design)

Comment (maliberty) [reply src/dbSta/src/dbSta.cc:557]: If you run report_cell_usage on two different modules then the metrics will collide.

Comment (gadfort) [reply src/dbSta/src/dbSta.cc:557]: this is true to many commands, I change the metrics stage in my script before each call, but we could change the metrics prefix if you have a suggestion.

Comment (gadfort) [reply src/dbSta/src/dbSta.cc:557]: for the top module would you drop the `__in_module`?

Comment (maliberty) [reply src/dbSta/src/dbSta.cc:557]: The top module is the same as whole design, ie design__instance__count.

**Comment by maliberty** [comment]
It would be nice to have a test for this.  Do you mind adding a small one?

**Comment by gadfort** [comment]
@maliberty I figured it out, test added.

**Review by maliberty — APPROVED** [review]

**Pull request merged**